Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
+      activemodel (= 6.1.3)
+      activesupport (= 6.1.3)
+    activesupport (6.1.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-      zeitwerk (~> 2.2, >= 2.2.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
     builder (3.2.4)
-    concurrent-ruby (1.1.7)
+    concurrent-ruby (1.1.8)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.7 → 1.1.8

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.1.7...v1.1.8).

These are the individual commits:
- (c4cbc96) [Do not iterate over hash which might conflict with new pair …](https://github.com/ruby-concurrency/concurrent-ruby/commit/c4cbc968c55e5b983dae953095761896220c46d1)
- (5eaecdc) [Fix race condition in TVar for stale reads, fixes #885](https://github.com/ruby-concurrency/concurrent-ruby/commit/5eaecdc9102d54c947b04f1d53833db4d1cde3f8)
- (e718a78) [Fix race condition in TVar for stale reads, fixes #885](https://github.com/ruby-concurrency/concurrent-ruby/commit/e718a78eb68597868c84518a17de94e527bb7841)
- (31669c6) [Merge pull request #886 from ruby-concurrency/stale-reads-ra…](https://github.com/ruby-concurrency/concurrent-ruby/commit/31669c6733d1ca7a3f5463fc56328320b9f374e7)
- (63a6907) [Merge pull request #884 from ruby-concurrency/thread-local](https://github.com/ruby-concurrency/concurrent-ruby/commit/63a690770cebb836d0c66c3022327d5399f8894e)
- (c951c33) [Clarify opacity of TVar in the current implementation](https://github.com/ruby-concurrency/concurrent-ruby/commit/c951c33e1050144ebf2c1b5f26544a9b6aedb365)
- (a5076bf) [Merge pull request #890 from ruby-concurrency/clarify-opacit…](https://github.com/ruby-concurrency/concurrent-ruby/commit/a5076bf9e977b290cce5a65b7b72810562bca295)
- (eb400d3) [Bump version 1.1.8](https://github.com/ruby-concurrency/concurrent-ruby/commit/eb400d3c1c62d02afc3ae79db8deed86c0167bb2)
- (b47d294) [Update documentation](https://github.com/ruby-concurrency/concurrent-ruby/commit/b47d29412b05a4a502fad66a2ac3699013178e89)


</details>